src/main/java/com/dokev/gold_dduck/gift/converter/GiftConverter.java

```diff
@@ -0,0 +1,34 @@
+package com.dokev.gold_dduck.gift.converter;
+
+import com.dokev.gold_dduck.gift.domain.Gift;
+import com.dokev.gold_dduck.gift.domain.GiftItem;
+import com.dokev.gold_dduck.gift.dto.GiftDto;
+import com.dokev.gold_dduck.gift.dto.GiftItemDto;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.springframework.stereotype.Component;
+
+@Component
+public class GiftConverter {
+
+    private static final int EMPTY = 0;
+
+    public GiftDto convertToGiftDto(Gift gift) {
+        List<GiftItem> giftItems = gift.getGiftItems();
+        List<GiftItemDto> giftItemDtos = giftItems.stream()
+                .map(this::convertToGiftItemDto)
+                .collect(Collectors.toList());
+
+        if (gift.getItemCount().isEmpty()) {
+            return new GiftDto(gift.getId(), gift.getCategory(), EMPTY, giftItemDtos);
+        }
```

Comment:
GiftDto를 만들때 getItemCount().isEmpty() true 면 Empty를 넣게 하셨는데 0일 때 익셉션을 발생시키면 좋겟지만 0일때 그냥 EMPTY를 넣는거라면 사실 ItemCount가 선물 아이템 갯수를 뜻하는 거니까 giftItemDtos.size()를 넣으면 조건문이랑 EMPTY가 없어도 될 것 같습니다.

```suggestion
        List<GiftItem> giftItems = gift.getGiftItems();
        List<GiftItemDto> giftItemDtos = giftItems.stream()
                .map(this::convertToGiftItemDto)
                .collect(Collectors.toList());

            return new GiftDto(gift.getId(), gift.getCategory(), giftItemDtos.size() , giftItemDtos);
```